README.md

```diff
@@ -97,141 +93,119 @@ After you're finished please remove all the comments and instructions!
 
-To get a local copy up and running, follow these steps.
+Clone this repository to your desired folder:
 
-### Prerequisites
+```bash
+$ git clone https://github.com/DuaneDave/Blog-app.git
+$ cd blog_app
+```
 
-In order to run this project you need:
 
-<!--
-Example command:
 
-```sh
- gem install rails
-```
- -->
+### Prerequisites
 
-### Setup
+In order to work on this project, you need to have the following dependencies installed:
 
-Clone this repository to your desired folder:
+- [Ruby](https://www.ruby-lang.org/en/)
+- [Postgresql](https://www.postgresql.org/)
+- [Node.js](https://nodejs.org/en/)
+- [Yarn](https://yarnpkg.com/)
+- [Rails](https://rubyonrails.org/)
 
-<!--
-Example commands:
 
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
+```bash
+$ bundle install
 ```
---->
 
-### Usage
+To get all the gems required for the project
 
-To run the project, execute the following command:
-
-<!--
-Example command:
+Then, run
 
-```sh
-  rails server
+```bash
+$ npm install
 ```
---->
 
-### Run tests
-
-To run tests, run the following command:
+To install packages such as style linters
 
-<!--
-Example command:
+To check linters locally, use
 
-```sh
-  bin/rails test test/models/article_test.rb
 ```
---->
+bash
+$ rubocop
+$ npx stylelint "**/*.{css,scss}"
+```
 
-### Deployment
 
-You can deploy this project using:
 
-<!--
-Example:
+### Usage
 
-```sh
+To run the project, execute the following command:
 
 ```
- -->
+rails server
+```
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
 
-## 👥 Authors <a name="authors"></a>
+### Run tests
+
+To run tests, run the following command:
+  
+  ```sh
+    rspec ./spec/models
+    rspec ./spec/requests
+  ```
 
-> Mention all of the collaborators of this project.
 
-👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+## 👥 Authors <a name="authors"></a>
+
+## Author
 
-👤 **Author2**
+👤 **Obiebi David**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/aeh1707)
+- Twitter: [@twitterhandle](https://twitter.com/aeh1707)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abdelhadi-hireche/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FUTURE FEATURES -->
 
-## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add a blog]**
+- [ ] **[Delete a blog]**
```

Comment:
- [ ] Let me understand the purpose of these features. We should only add necessary links/instructions or commands.